test/meson.build

```diff
@@ -377,6 +377,7 @@ foreach dirname : [
         'TEST-84-STORAGETM',
         'TEST-85-NETWORK',
         'TEST-86-MULTI-PROFILE-UKI',
+        'TEST-87-EXTRA-FILE-DESCRIPTOR',
```

Comment:
You can make this a subtest of `TEST-23-UNIT-FILE` instead. We try to only add new top level tests for big features